pkg/doc/jose/encrypter_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jose
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/tink/go/keyset"
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/json"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes"
+	ecdhessubtle "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle"
+)
+
+func TestJWEEncrypt(t *testing.T) {
+	_, err := NewJWEEncrypt(nil)
+	require.Error(t, err, "NewJWEEncrypt should fail with empty recipientPubKeys")
```

Comment:
Consider checking for the specific error instead of just expecting any error

Reply:
I don't wanna bloat testing when specific errors. Refactoring becomes much tedious when checking for specific errors.